tests/ro-crate-metadata.json

```diff
@@ -0,0 +1,457 @@
+                }
+            ]
+        },
+        {
+            "@id": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/tests/CWL/wf/trimming_and_qc.cwl",
+            "@type": [
+                "File",
+                "SoftwareSourceCode",
+                "ComputationalWorkflow"
+            ],
+            "attachment": [
+                {
+                    "@id": "exe/fastqc.cwl"
+                },
+                {
+                    "@id": "exe/trimmomatic_pe.cwl"
+                }
+            ],
+            "description": {
+                "@id": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/README.md"
+            },
+            "input": [
+                {
+                    "@id": "#fastq_1"
+                },
+                {
+                    "@id": "#fastq_2"
+                }
+            ],
+            "name": "CWL_trimming_and_qc",
+            "output": [
+                {
+                    "@id": "#ERR034597_1.small.fq.trimmed.1P.fq"
+                },
+                {
+                    "@id": "#ERR034597_1.small.fq.trimmed.1U.fq"
+                },
+                {
+                    "@id": "#ERR034597_1.small.fq.trimmed.2P.fq"
+                },
+                {
+                    "@id": "#ERR034597_1.small.fq.trimmed.2U.fq"
+                },
+                {
+                    "@id": "#ERR034597_1.small_fastqc.html"
+                },
+                {
+                    "@id": "#ERR034597_2.small_fastqc.html"
+                },
+                {
+                    "@id": "stdout.log"
+                },
+                {
+                    "@id": "stderr.log"
+                }
+            ],
+            "programmingLanguage": {
+                "@id": "https://w3id.org/workflowhub/workflow-ro-crate#cwl"
+            },
+            "version": "1.0.0",
+            "yevisId": "c13b6e27-a4ee-426f-8bdb-8cf5c4310bad"
+        },
+        {
+            "@id": "https://w3id.org/cwl/vv1.0/",
+            "@type": [
+                "WebPage"
+            ]
+        },
+        {
+            "@id": "https://www.commonwl.org/",
+            "@type": [
+                "WebPage"
+            ]
+        },
+        {
+            "@id": "https://w3id.org/workflowhub/workflow-ro-crate#cwl",
+            "@type": "ComputerLanguage",
+            "alternateName": "CWL",
+            "identifier": {
+                "@id": "https://w3id.org/cwl/vv1.0/"
+            },
+            "name": "Common Workflow Language",
+            "url": {
+                "@id": "https://www.commonwl.org/"
+            },
+            "version": "v1.0"
+        },
+        {
+            "@id": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/README.md",
+            "@type": [
+                "WebPage"
```

Comment:
Since it's used as the value of a [description](https://schema.org/description) (in the workflow entity), the type here should be [TextObject](https://schema.org/TextObject)